pkg/apis/common/resource.go

```diff
@@ -0,0 +1,30 @@
+package common
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+// PartialResource represents a linked resource in GS
+type PartialResource struct {
+	Identifier string `json:"identifier,omitempty"`
+	Name       string `json:"name,omitempty"`
+}
```

Comment:
this not only exists in GS but is a pattern in our APIs quite often - just strip the "in GS" from the docs